### PR TITLE
Big refactor, many minor fixes

### LINKED ISSUE #395: [maint] Ubuntu Uses Snaps by Default
Need to refactor, or at least give giant warnings on [Linux LAN](https://docs.start9.com/latest/user-manual/connecting/connecting-lan/lan-os/lan-linux) of this fact, and alternate steps.

### LINKED ISSUE #440: [update] New Password Reset Flow
Now within the UI, gut and update

### PR DESCRIPTION
**Pull request by kn0wmad**
- Adds OS icons for device guides
- Refactor to simpler layout
- Deprecate "Learn" section
- New flow to add cert for all connections as priority
- Closes #395 
- Closes #440

### CODE REVIEW

**Comment by kn0wmad** [comment]
Currently missing Reset Password flow, which @gStart9 will be adding

File: site/source/guides/index.rst [hunk 1]
```diff
@@ -12,5 +12,6 @@ Guides for integrating your client devices with your Start9 server, building you
   service-guides/index
   upgrade-hardware/index
   flashing/index
+  change-password
```
Comment (gStart9) [inline site/source/guides/index.rst:15]: Should this exist?  Change password seems like a guide but the actual page for this guide that I added to is under user-manual/overview.  Adding this entry results in this:
image
Which also currently also exists under Overview:
image

**Comment by elvece** [comment]
Display issue on support index page:

Screenshot 2023-07-27 at 5 40 04 PM

File: site/source/guides/index.rst (hunk 1 above)
Comment (elvece) [reply site/source/guides/index.rst]: I think this organization makes sense. We need to have a better UX in StartOS around prompting these features, especially backups.

**Review by elvece — CHANGES_REQUESTED** [review]
Needs a rebase and resolution of one display bug. I like the organization.

**Comment by kn0wmad** [comment]
> Display issue on support index page:
> Screenshot 2023-07-27 at 5 40 04 PM

Fixed

**Review by elvece — APPROVED** [review]

**Pull request merged**